### ISSUE TITLE
New Hackage release needed

### ISSUE THREAD

**Issue by osense** [m4dc4p/haskelldb#30]
The current version at http://hackage.haskell.org/package/haskelldb doesn't include the **GHC 8.0** compatibility patch.

I just patched it myself, only to find out it's already been done -.-

**Comment by m4dc4p** [comment]
I recommend you build from source. I don't know when I'll find time to put a new release up. My apologies.

**Comment by osense** [comment]
Fair enough, I mostly opened the issue as a way to signal to other people who might end up here that the package on Hackage is outdated.

**Comment by m4dc4p** [comment]
👍
